packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -200,6 +227,16 @@ export const CommentsItem = ({
   const [replyFormIsOpen, setReplyFormIsOpen] = useState(false);
   const [showEditState, setShowEditState] = useState(false);
   const [showParentState, setShowParentState] = useState(showParentDefault);
+  const [activePotentialComment, setActivePotentialComment] = React.useState<0|1|2>(0)
+  const [adminTriedToOptOut, setAdminTriedToOptOut] = React.useState<string|null>(null)
+  const potentialCommentBodies = useMemo(() => ([...(comment.doppelComments ?? []), comment])
+    .map(a => ({
+      content: 'commentId' in a ? a.content : (a.contents?.html ?? ""),
+      score: Math.random(),
+      doppelId: 'commentId' in a ? a._id : null
+    }))
```

Comment:
s/a/doppelComment, or something

Reply:
also I get the impulse for purity but probably too complicated for inline assignments; I'd assign to variables:

```
const content = 'commentId' in ???
  ? ???.content
  : ???.contents?.html ?? '';
```

Reply:
Also is `score: Math.random()` intentional?

Reply:
Oh, it's just used as a randomized sort.  Probably use `lodash/shuffle` instead?